tests/test_stats.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""Tests for the vector submodule."""
+
+import pytest
+import numpy as np
+import xarray as xr
+
+from hydromt import stats
+
+
+def test_geo(obsda, simda):
+    assert stats.bias(simda, obsda).values.round(2) == 5.0
```

Comment:
For these type of statistics two decimals is not a lot for comparison, small differences will not be noted. 
I suggest to use np.isclose with much more decimals or round with at least 5 decimals.